src/mds/CInode.cc

```diff
@@ -4139,6 +4139,7 @@ void CInode::encode_cap_message(const ref_t<MClientCaps> &m, Capability *cap)
   m->size = i->size;
   m->truncate_seq = i->truncate_seq;
   m->truncate_size = i->truncate_size;
+  m->fscrypt_file = pi->fscrypt_file;
```

Comment:
For the `fscrypt_auth`, for now I don't will it be used in client side when updating the caps.

Will it make sense to send it to kclient here in future uses ?